### PR TITLE
Add left/right keybindings to dashboard view.

### PR DESCRIPTION
**Pull request by ddimensia**
I added the ability to capture the 'left'/'right' keyboard keypresses on the dashboard view so that it could mimic pressing the left/right nav buttons respectively.

### CODE REVIEW

**Comment by ddimensia** [comment]
I guess I can't set label/milestone.  Too used to working with PR's against repos I own.

**Comment by brentpicasso** [comment]
addresses #1028

**Comment by ryandoherty** [comment]
Looks good to me and works! 👍

**Comment by ryandoherty** [comment]
Oops, spoke too soon. We follow the PEP8 standards, see our CONTRIBUTING doc for details. I will comment on specific things that should be updated.

File: autosportlabs/racecapture/views/dashboard/dashboardview.py [hunk 1]
```diff
@@ -117,6 +119,13 @@ def on_mouse_pos(self, x, pos):
             self._got_activity()
             self.ids.preferences_button.brighten()
         return False
+
+    def on_key_down(self, window, key, *args):
+        if key==Keyboard.keycodes.get('left',-1):
```
Comment (ryandoherty) [inline autosportlabs/racecapture/views/dashboard/dashboardview.py]: PEP8 styling would have this line look like this:

`if key == Keyboard.keycodes.get('left', -1):`

File: autosportlabs/racecapture/views/dashboard/dashboardview.py [hunk 2]
```diff
@@ -117,6 +119,13 @@ def on_mouse_pos(self, x, pos):
...
+          self.on_nav_left()
```
Comment (ryandoherty) [inline autosportlabs/racecapture/views/dashboard/dashboardview.py]: Needs 4 spaces for indentation.

File: autosportlabs/racecapture/views/dashboard/dashboardview.py [hunk 3]
```diff
@@ -117,6 +119,13 @@ def on_mouse_pos(self, x, pos):
...
+        elif key==Keyboard.keycodes.get('right',-1):
+          self.on_nav_right()
+        return False
```
Comment (ryandoherty) [inline autosportlabs/racecapture/views/dashboard/dashboardview.py]: Is a return value needed?

**Comment by ddimensia** [comment]
Fixed the formatting issues, forgot to run the code through a formatter.  At some point it would be nice if the repo had a pylint config that the code was passed through as part of a Travis build.  

The reason for the return value is that it indicates if the key input should be propagated to on_keyboard or not.  Arguably it should maybe return True.

File: autosportlabs/racecapture/views/dashboard/dashboardview.py [hunk 4]
```diff
@@ -19,6 +19,7 @@
 from kivy.clock import Clock
 from kivy.uix.modalview import ModalView
 import os
+import traceback
```
Comment (ryandoherty) [inline autosportlabs/racecapture/views/dashboard/dashboardview.py]: Debug code left over?

**Comment by ryandoherty** [comment]
RE: the return value, I don't have an opinion either way. I think it's fine to leave it for now.

**Comment by ddimensia** [comment]
Removed debug import...  I just amended the commit rather than have a bunch of extra commits.  I was just verifying that the on_key_down as called as expected w/ respect to the return value.

**Comment by ryandoherty** [comment]
Awesome! Can you update the CHANGELOG to have this new functionality listed?

**Comment by ddimensia** [comment]
Yep, will do, probably won't get to it until tonight though.

**Comment by ryandoherty** [comment]
No problem, thanks!

**Comment by ddimensia** [comment]
Will need to reopen against 1.6.0 after updating the CHANGELOG

**Pull request closed without merging**